aten/src/ATen/native/native_functions.yaml

```diff
@@ -1930,7 +1930,11 @@
 
 - func: max.dim_max(Tensor self, int dim, bool keepdim=False, *, Tensor(a!) max, Tensor(b!) max_values) -> (Tensor(a!) values, Tensor(b!) indices)
 
-- func: max_values(Tensor self, int[1] dim, bool keepdim=False) -> Tensor
+- func: max_values(Tensor self, int[1] dim=[], bool keepdim=False) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+
+- func: amax(Tensor self, int[1] dim=[], bool keepdim=False) -> Tensor
```

Comment:
I think you should add the `, *, out` signature even in this first PR. It's straightforward to add. (Same with a derivative.)